subhub/api/webhooks/stripe/abstract.py

```diff
@@ -1,18 +1,12 @@
 import logging
-from abc import ABC, abstractmethod
-import requests
 
+from abc import ABC, abstractmethod
 import requests
 from subhub.api.webhooks.routes.pipeline import RoutesPipeline
 from subhub.cfg import CFG
-from subhub import secrets
-
-logger = logging.getLogger("webhook_abstract")
-log_handle = logging.StreamHandler()
-log_handle.setLevel(logging.INFO)
-logformat = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
-log_handle.setFormatter(logformat)
-logger.addHandler(log_handle)
+from subhub import log
+
+logger = log.setup(logging.getLogger("webhook_route"))
```

Comment:
Why is this "webhook_route" instead of "abstract"

Reply:
When I was finishing off the updated, I didn't go over the work that was already done. I will update this :)